### PR TITLE
Tweak local copy of project code of conduct

### PR DESCRIPTION
**Pull request by nekomeowww**
## Changes

1. Fix typos for several words
2. Use mailto for email addresses for better user experience and aligned with semantic anchor links
3. There are tones of unnecessary trailing spaces in the file, removed them

### CODE REVIEW

**Review by aj11anuj — COMMENTED** [review]
LGTM, 👍

File: content/en/community/static/cncf-code-of-conduct.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!-- Do not edit this file directly. Get the latest from
      https://github.com/cncf/foundation/blob/main/code-of-conduct.md -->
```
Comment (sftim) [inline content/en/community/static/cncf-code-of-conduct.md:2]: > ```
> 
> ```

**Review by sftim — COMMENTED** [review]
/hold

**Comment by sftim** [comment]
/lgtm cancel
See https://github.com/kubernetes/website/pull/43737#discussion_r1375863683

**Comment by sftim** [comment]
/retitle Tweak local copy of project code of conduct

**Comment by sftim** [comment]
@nekomeowww I'm afraid you must suggest changes to the original file, and not to the copy here.

**Comment by nekomeowww** [comment]
Ah, sorry that I missed this comment. I will close this one and create Pull Request for the original files.

Thanks for reviewing guys!

**Comment by nekomeowww** [comment]
Created at https://github.com/cncf/foundation/pull/659

**Pull request closed without merging**